connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/errors/BadRequestException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.apache.kafka.connect.runtime.rest.errors;
+
+import javax.ws.rs.core.Response;
+
+public class BadRequestException extends ConnectRestException {
+
+    public BadRequestException(String message) {
+        super(Response.Status.BAD_REQUEST, message);
```

Comment:
It's nice if we can be more specific about the type of exception and include an error code. `ConnectExceptionMapper` will then convert the body of the error response to the standard format and include the more specific status code which gives folks a way to figure out why the error is happening and potentially handle different ones programmatically. We've used 5-digit error codes to give more info beyond the HTTP status code. (Although I think we, admittedly, have not managed to take advantage of this yet; we do something similar in Confluent's REST proxy/schema registry.)